src/rulesets/__tests__/merger.unit.ts

```diff
@@ -142,7 +143,7 @@ describe('Rulesets merger', () => {
       },
     });
 
-    expect(ruleset).toHaveProperty('rules.test.severity', -1);
+    expect(ruleset).toHaveProperty('rules.test.severity', DiagnosticSeverity.Error);
```

Comment:
My apologies, but I had to make this change in order to make the following case work properly
```
extends: [[spectral:oas2, off]]
rules:
  operation-operationId-unique: true
```
Actually, if you dive into it, we have two causes of a severity change:
- the rule severity (the severity you put on rule)
- the ruleset severity (the severity you put on ruleset via extends)

This design a flaw:
spectral:oas2 *turns off* rules of spectral:oas, so severity of  `operation-operationId-unique` is set to -1
Then, we inherit severity value, therefore it's -1.

Now that I think of it, we could potentially bring back the `enabled` property to handle `off` just for the sake of normalization process, but this is still very tricky.

Let me know what your thoughts are @marbemac.

Reply:
Interesting - I don't think we should block the release for this. Folks can put in a severity for now, rather than true.

Reply:
Well, I do support `true` and `false`, but to a certain extent.
When you provide `true` we take the severity from the original/first rule definition.
I believe this is a safe assumption, for now, to make, if one wants to have a particular severity, one can always set it.